src/app/pages/data-protection/cloudsync/cloudsync-form/cloudsync-form.component.ts

```diff
@@ -687,7 +694,18 @@ export class CloudsyncFormComponent implements FormConfiguration {
     // When user interacts with direction dropdown, change transfer_mode to COPY
     this.formGroup
       .get('direction')
-      .valueChanges.pipe(filter(() => this.formGroup.get('transfer_mode').value !== TransferMode.Copy))
+      .valueChanges
+      .pipe(tap((direction: string) => {
+        const xattr_fc = entityForm.fieldConfig.find((config: any) => config.name === 'xattr');
```

Comment:
`xattr_fc` => `xattrFc` or even `xAttrControl`

`config` is probably of type `FieldConfig`